ghub.el

```diff
@@ -379,7 +379,8 @@ called after the next request has finished.  Use the function
   (and (assq 'next (ghub-response-link-relations req))
        (or (ghub--retrieve nil req) t)))
 
-(cl-defun ghub-wait (resource &optional duration &key username auth host)
+(cl-defun ghub-wait (resource &optional (duration 64)
+                              &key username auth host (forge 'github))
```

Comment:
I think FORGE needs no default and specifying defaults like this is kinda dangerous a caller might provide nil as the value and expect the default to be used. So just stick to setting DURATION as before.